<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetrievalWidget</class>
 <widget class="QWidget" name="RetrievalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>326</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="retrievalTab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="loadSceneListPushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>load our dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadPartDataPushButton">
         <property name="text">
          <string>load part-level dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="loadPoseDataPushButton">
           <property name="text">
            <string>load shape2pose dataset</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="snapModeLayout">
           <item>
            <widget class="QRadioButton" name="modelSnapRadioButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>model</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="poseSnapRadioButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>pose</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="minimumSize">
          <size>
           <width>326</width>
           <height>0</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Descriptors</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QPushButton" name="DoRetrieval_ICON">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>ICON</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="DoRetrieval_ISET">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>ISET</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="DoRetrieval_IBSH">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>IBSH</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Depth:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="depthSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="DoRetrieval_POSE">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>POSE</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="DoRetrieval_LFD">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>LFD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>combWeight(ICON):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="combWeightSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="DoRetrieval_LFDICON">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>LFD+ICON</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Part-level descriptors</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="DoRetrieval_GEO">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>GEO</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>combWeight(ICON):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="partCombWeightSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="DoRetrieval_GEOICON">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>GEO+ICON</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>PR-curve</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="prLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QRadioButton" name="iconRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>ICON</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="isetRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>ISET</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QRadioButton" name="lfdRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>LFD</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QRadioButton" name="lfd_iconRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>LFD+ICON</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QRadioButton" name="geoRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>GEO</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QRadioButton" name="geo_iconRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>GEO+ICON</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="ibshRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>IBSH</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QRadioButton" name="poseRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>POSE</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="prLayout_2">
              <item>
               <widget class="QRadioButton" name="prShapeRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>PR-Curve for Shape</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_2</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="prClassRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>PR-Curve for Category</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_2</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>loadSceneListPushButton</zorder>
      <zorder>tableWidget</zorder>
      <zorder>groupBox_5</zorder>
      <zorder>groupBox</zorder>
      <zorder>loadPartDataPushButton</zorder>
      <zorder>groupBox_3</zorder>
      <zorder></zorder>
      <zorder>loadPoseDataPushButton</zorder>
      <zorder>horizontalSpacer</zorder>
     </widget>
     <widget class="QWidget" name="resultTab">
      <attribute name="title">
       <string>Results</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="retrievalWidget">
         <property name="minimumSize">
          <size>
           <width>326</width>
           <height>0</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="loadRetrievalPushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Load retrieved scenes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="returnRetrievalSpinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
